### DISCUSSION TITLE
pt/guide/free

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#767] [Announcements]
# pt/guide/free

Como receber a licença grátis do Korepi. Passo 1: Encontre o canal ★⋅micah-bot-verify⋅★ link direto (https://discord.com/channels/1069057220802781265/1203687333107335198); img.p...

https://korepi.com/pt/guide/free.html

**Comment by levyvix** [comment]
Muito bom tutorial! Ótimo trabalho.
